tests/e2e/vsan_stretched_cluster.go

```diff
@@ -447,6 +486,7 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 			fds.hostsDown = nil
 		}
 
+		time.Sleep(2 * time.Minute)
```

Comment:
I am not sure if 2 mins or 3 mins or whatever 5 mins of static sleep will solve the issue, when we trigger E2E tests in the Jenkins job, multiple times testbed will be going back and forth in a healthy/unhealthy state, see if we can think of some other optimised approach of making sure that testbed is in a healthy state.